### PR TITLE
Update Lossy_checkpoint_restart.md

### PR DESCRIPTION
**Pull request by Franckcappello**
Hope the citation will work

### CODE REVIEW

**Comment by pancetta** [comment]
Sorry, the citation does not work. What are you trying to cite here? 

Check out e.g. [this project](https://github.com/JLESC/jlesc.github.io/blob/source/_projects/exec_interfer_project.md) to see how it should look like.

**Review by pancetta — CHANGES_REQUESTED** [review]
Citation is not working, please fix (see comment).

File: _projects/Lossy_checkpoint_restart.md [hunk 1]
```diff
@@ -34,19 +34,26 @@ In contrary to previous research that concentrated on few applications, we focus
...
+
 ## Visits and meetings
 
 Franck Cappello visits UIUC almost every week. We have a 30 minutes to 1 hour meeting almost each time.
+Jon Calhoun did an internship of 11 weeks at ANL.
 
 ## Impact and publications
 None yet.
-{% bibliography --cited --file jlesc.bib %}
-
```
Comment (pancetta) [inline _projects/Lossy_checkpoint_restart.md]: Please leave this line untouched.

File: _projects/Lossy_checkpoint_restart.md [hunk 2]
```diff
@@ -34,19 +34,26 @@ In contrary to previous research that concentrated on few applications, we focus
...
+\cite{Calhoun17}
```
Comment (pancetta) [inline _projects/Lossy_checkpoint_restart.md]: Where is this reference located?

Comment (pancetta) [reply _projects/Lossy_checkpoint_restart.md]: Also, see e.g. https://github.com/JLESC/jlesc.github.io/blob/source/_projects/exec_interfer_project.md for an example.

**Comment by Franckcappello** [comment]
I fixed the citation. Hope this works now

**Comment by pancetta** [comment]
Getting closer: You need to add the bibtex entry to the [jlesc.bib](https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib) file. Also, please restore line 43 with the `{% bibliography --cited --file jlesc.bib %}` code.

**Comment by Franckcappello** [comment]
Getting closer: You need to add the bibtex entry to the jlesc.bib<https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib> file.

I did that. But the editor did not ask for a pull request.



Also, please restore line 43 with the {% bibliography --cited --file jlesc.bib %} code.

Ok, will do that.

Franck

**Comment by pancetta** [comment]
Hmm, weird. Could you try to edit the jlesc.bib file ([this one](https://github.com/Franckcappello/jlesc.github.io/blob/patch-5/_bibliography/jlesc.bib)) again, please?

**Comment by Franckcappello** [comment]
Same thing.
I only have a green button saying: “commit change”. I clicked on it but these is no pull request button.
Franck

**Comment by pancetta** [comment]
No, this did the trick! The last commit did not create a PR, since it changed the current one. You're all set!

**Pull request merged**